app/helper.js

```diff
@@ -57,6 +57,12 @@ class YoutubeGrabberHelper {
   }
 
   async parseChannelVideoResponse(response, channelId, channelIdType) {
+    if (typeof (response.data[1].response.alerts) !== 'undefined') {
+      return {
+        alert: response.data[1].response.alerts[0].alertRenderer.text.simpleText
```

Comment:
You can see when you pull the message via `response.data[1].response.alerts[0].alertRenderer.text.simpleText`
`response.alerts` is an array of object not strings
Which is why the "alert message/text" should be named `alertMessage`/`alertText`